### PR TITLE
chore: Revised logo (new color), w/CSS for vert alignment

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

We wanted to update the color of the superset logo. I'm not sure why the existing one had a white stroke, so I used a rasterization of the SVG logo from `superset-frontent/branding`. 

Due to the slightly different size/aspect ratio (which... I actually like), the vertical alignment was a hair off. I added some styles (using Emotion, rather than adding to the LESS codebase) to vertically center logos of any size.

Note that the styles (and thus these screenshots) are for the React menu. I don't _think_ the FAB-generated nav bar is viewable any more, unless I'm mistaken. @nytai could you confirm?

Areas of feature-creep I'm ignoring for this PR, but might circle back to:
1) Menu.jsx -> Menu.tsx
2) Removing any styles/jinja templates relating to the old menu that are not visible in superset anymore 
3) Now that Emotion has entered the picture, there are probably various LESS styles that can be migrated in.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:
image

After:
image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by nytai** [comment]
@rusackas I believe the FAB generated menu is still used on certain route, eg the FAB generated ones for example (try `/databaseview/list/` or `/csvtodatabaseview/form`). We really should migrate off the old FAB menu, it's getting pretty confusing. I'll take a look and see what's possible in the coming days.

**Comment by mistercrunch** [comment]
We still use the FAB navbar in some places but it looks ok!
Screen Shot 2020-07-03 at 7 28 05 AM
Screen Shot 2020-07-03 at 7 27 56 AM

About `black` failing, you should install pre-commit! https://github.com/apache/incubator-superset/blob/master/CONTRIBUTING.md#git-hooks

I'll stack a commit to try and get CI to pass

File: superset-frontend/images/superset-logo-horiz.svg [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <path d="M73.79,15.23C67.32,15.23 61.36,18.87 55.6,25.23C49.94,18.77 43.88,15.23 37.11,15.23C25.9,15.23 17.72,23.23 17.72,34C17.72,44.77 25.9,52.67 37.11,52.67C44,52.67 49.34,49.44 55.3,43C61.06,49.46 66.92,52.69 73.79,52.69C85,52.67 93.18,44.8 93.18,34C93.18,23.2 85,15.23 73.79,15.23ZM37.19,41.37C32.44,41.37 29.61,38.24 29.61,34.1C29.61,29.96 32.44,26.74 37.19,26.74C41.19,26.74 44.46,29.96 48,34.3C44.66,38.34 41.13,41.37 37.19,41.37ZM73.45,41.37C69.51,41.37 66.18,38.24 62.64,34.1C66.28,29.76 69.41,26.74 73.45,26.74C78.2,26.74 81,30 81,34.1C81,38.2 78.2,41.37 73.45,41.37Z" style="fill:rgb(72,72,72);fill-rule:nonzero;"/>
+    <path d="M63.74,50L71.28,41C68.28,40.1 65.51,37.4 62.64,34.05L55.3,43C57.703,45.788 60.556,48.154 63.74,50Z" style="fill:rgb(32,167,201);fill-rule:nonzero;"/>
+    <g id="Main">
+        <g id="Superset">
+            <g id="Full-Lockup-With-Text">
+                <g id="Group-7">
+                    <g id="Group-17">
+                        <g id="Superset-Copy">
+                            <g>
```
Comment (etr2460) [inline superset-frontend/images/superset-logo-horiz.svg:28]: let's get rid of some of these unnecessary `g` tags?

File: superset-frontend/src/components/Menu/Menu.jsx [hunk 2]
```diff
@@ -51,11 +52,19 @@ const propTypes = {
   }).isRequired,
 };
 
+const Styles = styled.header`
```
Comment (etr2460) [inline superset-frontend/src/components/Menu/Menu.jsx:55]: Maybe a name like `StyledHeader` might be more clear than `Styles`?

File: superset-frontend/images/superset-logo-horiz.svg [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (etr2460) [inline superset-frontend/images/superset-logo-horiz.svg:1]: Where is the SVG used? I don't see it being used anywhere

**Review by etr2460 — COMMENTED** [review]
The new logo as a "TM" on it. Is that necessary? I don't see a lot of other websites add a TM to their logo when it's used in the navbar

File: superset-frontend/images/superset-logo-horiz.svg (hunk 3 above)
Comment (rusackas) [reply superset-frontend/images/superset-logo-horiz.svg:1]: It's basically vestigial. The "thing I wanted to try" was editing this SVG file to use "currentColor" for the blue fill areas, such that the SVG inherits the brandColor variable. Then if people wanted to restyle their superset instance to use orange (or whatever) in the brand color, it'd be reflected in the logo.

However, that basically requires building in some JSX that depends on the logo using SVG. I'm sure lots of people are using JPG/PNG logos, so a big revamp around this suddenly out of scope.

TL;DR: it's a bit vestigial for now, but I hope to circle back and make good use of it. I can remove it from the PR if it's a bother.

File: superset-frontend/images/superset-logo-horiz.svg (hunk 1 above)
Comment (rusackas) [reply superset-frontend/images/superset-logo-horiz.svg]: Meh... I'll just delete it. I'll keep the SVG trimming in mind if/when it makes a comeback.

**Comment by rusackas** [comment]
> The new logo as a "TM" on it. Is that necessary? I don't see a lot of other websites add a TM to their logo when it's used in the navbar

Removed.

image

**Review by mistercrunch — APPROVED** [review]

**Review by etr2460 — APPROVED** [review]

File: superset-frontend/src/components/Menu/Menu.jsx (hunk 2 above)
Comment (ktmud) [reply superset-frontend/src/components/Menu/Menu.jsx]: I think it's fine to use `Styles`. This is for internal use, kind of like the `<style>` tag in Vue's [Single File Component](https://vuejs.org/v2/guide/single-file-components.html).
